resources/qml/pages/MessagingSettingsPage.qml

```diff
@@ -77,5 +78,38 @@ Page {
             }
         }
 
+        TextSwitch {
+            id: compressImagesSwitch
+            checked: shmoose.settings.CompressImages
+            text: qsTr("Compress images larger than")
+            onClicked: {
+                shmoose.settings.CompressImages = compressImagesSwitch.checked;
+                maxCompressedImageSizeSlider.enabled = compressImagesSwitch.checked;
+            }
+        }
+
+        Slider {
+            id: maxImageSizeSlider
+            enabled: shmoose.settings.CompressImages
+            width: parent.width
+            minimumValue: 100000
+            maximumValue: 10000000
+            stepSize: 100000
+            value: shmoose.settings.MaxImageSize
+            valueText: value/1000 + qsTr(" KB")
+
+            onValueChanged: {
```

Comment:
Is there is also something like 'onReleased'? My old test device is a Jolla1. With this, you see the performance issue on constantly update the values during pressing and sliding for writing config values to flash.

Reply:
Indeed, there is no such onReleased, at least in the documentation. Yes, it feels better now on the UI.